### DISCUSSION TITLE
Chart version changed from 0.3.1 to 0.24.0

### DISCUSSION THREAD

**Discussion by maartengo** [tchiotludo/akhq#1455] [General]
Hey, just wondering if the change from https://github.com/tchiotludo/akhq/commit/a0d2ec6592485bed7b50e587280e9af0a979fa64 was supposed to update the chart version from 0.3.1 to 0.24.0?
If it wasn't supposed to happen then I'll wait with updating until a new chart version is released.

**Comment by tchiotludo** [comment]
I have release a new version of AKHQ yesterday and I decided to align helm charts version with AKHQ version
